src/code/circuit_python/motor.md

```diff
@@ -0,0 +1,232 @@
+# Spin a Wheel
+
+In the previous tutorial we used some simple functions to turn a light
+on and off.  Lets use some more advanced code to now control a motor.
+In this program we'll make use of a
+[VEX MC-29 motor controller](https://www.vexrobotics.com/276-2193.html).
+These motor controllers use a
+[Pulse Width Modulation (PWM)](https://en.wikipedia.org/wiki/Pulse-width_modulation)
+signal to determine direction and magnitude of the motor rotation they
+control.
+
+For this program, we're going to use two new modules: `pwmio` and
+`adafruit_motor`.
+
+## The pwmio Module
+
+PWM is a simple way to send an analog signal over a digital pin. It is
+commonly used for controlling motors, servos, and other actuators. The
+`pwmio` module provides the `PWMOut` class to efficiently generate PWM
+signals on a pin using our microcontroller's built-in timers. There are
+two important values for us to control with a `PWMOut` object: frequency
+and duty cycle.
+
+Frequency is the number of times each second that the signal changes from
+low to high and back to low. This is measured in Hertz (Hz).
+
+Duty Cycle is the fraction of the total pulse time where the signal is
```

Comment:
Earlier you reference logic levels with true and false, but here high and low.  The relationship between high/low, true/false, and 3.3v/0v needs to be established.

Reply:
The hello lesson  connects these terminologies, albeit briefly. Do you think this needs to be expanded? https://github.com/barulicm/gizmo-platform.github.io/blob/86608c911a1a94ed49c9f6585d135575fccb6024/src/code/circuit_python/hello.md?plain=1#L56